backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -35,17 +34,6 @@
   @JoinColumn(name = "test_event_id")
   private TestEvent testEvent;
 
-  @ManyToOne
-  @JoinFormula(
```

Comment:
This JoinFormula was the culprit for the poor performance of the `test_order` query. There are several other N+1 problems with this GraphQL query, but this at least is a start